pzsd_bot/__main__.py

```diff
@@ -21,7 +21,7 @@ async def on_ready():
 async def run_bot():
     try:
         async with bot:
-            bot.load_extensions("pzsd_bot.cogs")
+            bot.load_extensions("pzsd_bot.cogs", recursive=True)
```

Comment:
what's this do

Reply:
It loads all the classes in the cog directory. I've had to switch on the recursive flag now that I've organized the modules into sub folders.